molgenis-data/src/main/java/org/molgenis/data/meta/AttributeRepositoryDecorator.java

```diff
@@ -48,22 +48,6 @@ public void update(Stream<Attribute> attrs)
 	@Override
 	public void delete(Attribute attr)
 	{
-		// If compound attribute is deleted then change the parent of children to null
```

Comment:
Since ```AttributeRepositoryDecorator``` extends from ```AbstractRepositoryDecorator``` and doesn't have to do anything different for deletes the following methods can/should be removed:
```
public void delete(Attribute attr)
public void delete(Stream<Attribute> attrs)
public void deleteById(Object id)
public void deleteAll(Stream<Object> ids)
public void deleteAll()
```